### ISSUE TITLE
Evaluate Code

### ISSUE THREAD

**Issue by Gary-code** [dev-chauhan/PQG-pytorch#9]
It seems like that after I trained the model with our own dataset using the code from master branch, I can not evaluate the model on the test set unless switching to ori- branch. However, I don't want to waste 2days training the model. What can I do to evaluate the testing data in the master branch.

**Comment by Gary-code** [comment]
I check the code between the ori-code branch and the master branch. The checkpoint in the save model has a big difference. One save the encoder and generator. Another just simply save the model.

**Comment by dev-chauhan** [comment]
In master branch you can use `evaluate_scores` function to evaluate on your own data, if you have saved your model then you just have to generate para phrases and then you can pass generated para phrase and ground truth to `evaluate_scores` function which will return the scores w.r.t. your ground truth in data and generated para phrases.

**Comment by Gary-code** [comment]
> 

Thanks a lot.

**Comment by Gary-code** [comment]
Which model in the paper is the best. EDLPGS?

**Comment by dev-chauhan** [comment]
Its EDLPS in terms of Bleu scores
